src/components/map/MapContainer.tsx

```
@@ -0,0 +1,42 @@
+    if (!kakaoMapKey) {
+      console.error('Kakao Map API Key is missing');
+      return;
+    }
+
+    const script = document.createElement('script');
+    script.src = `//dapi.kakao.com/v2/maps/sdk.js?appkey=${kakaoMapKey}&autoload=false`;
+    script.async = true;
+    document.head.appendChild(script);
+
+    script.onload = () => {
+      if (window.kakao && window.kakao.maps) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)